extensions/amp-auto-ads/0.1/anchor-ad-strategy.js

```diff
@@ -70,6 +74,22 @@ export class AnchorAdStrategy {
       .includes(OPT_IN_STATUS_ANCHOR_ADS);
   }
 
+  /**
+   * Add custom amp-analytics emement
+   * @param {HTMLElement} adElement
+   */
+  addCustomAmpAnalytics_(adElement) {
+    if (this.customAnalytics_) {
+      const doc = this.ampdoc.win.document;
+      const customAmpAnalytics = createElementWithAttributes(
+        doc,
+        'amp-analytics',
+        this.customAnalytics_
+      );
+      adElement.appendChild(customAmpAnalytics);
```

Comment:
Extensions are not allowed to create analytics this way. Please use `insertAnalyticsElement` from `src/extension-analytics` instead. This sandboxes the inserted element for user privacy.

Reply:
@alanorozco I tried to use the `insertAnalyticsElement` to create a `amp-analytics`, however thereare no way of create a `amp-analytics` with url config, only with script config pattern, so I made a modification and added a optional `opt_useUrlConfig` parameter to `insertAnalyticsElement` function to be possible to create a  amp-analytics with url config pattern

Reply:
@alanorozco I changed the logic to fetch the configuration from the url config to keep the sandbox flow